std_msgs/Header header
string group_name
bool is_relative
geometry_msgs/Pose goal
float32 tolerance
---
uint8 SUCCEEDED=0
uint8 FAILED=1
uint8 result_status